src/interpreter.rs

```diff
@@ -67,6 +67,8 @@ impl Interpreter {
                 let rhs = self.interpret(&expression.rhs)?;
 
                 match expression.operator {
+                    // FIXME(javad): PLEASE FIX
```

Comment:
Please refrain from making personal comments in code, and explain exactly what needs changing so other people can also contribute.